docker-compose.yml

```diff
@@ -47,3 +47,17 @@ services:
   #     - ./data:/data
   #     - ./sites:/srv
   #   network_mode: "host"
+
+# # Optional: If you need ipv6, follow step 1 and 2 of https://github.com/nextcloud/all-in-one/blob/main/docker-ipv6-support.md first and then uncomment the below config in order to activate ipv6 for the internal nextcloud-aio network
+# # Inspired by https://github.com/mailcow/mailcow-dockerized/blob/master/docker-compose.yml
+# networks:
+#   nextcloud-aio:
+#     driver: bridge
```

Comment:
```suggestion
#     name: nextcloud-aio
#     driver: bridge
```

Reply:
see: https://docs.docker.com/compose/compose-file/compose-file-v3/#name-1

Reply:
okay! do we want to add external: true to the config or rather not?

Reply:
I think `external: true` is not needed

Reply:
apparently rather not